packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 2.9.3
 
 * Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
+* Added `setVideoPositionUpdatesInterval` to allow setting position updates periodic intervals
```

Comment:
Adds

https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changelog-style

Reply:
This is also missing a period, also per the style guide.

Reply:
And it needs to be rewritten to reflect what the change now does; this is describing the previous iteration of the PR.

Reply:
thanks for the link, it should fit now